source/pages/Tokens/CustomToken.tsx

```diff
@@ -104,85 +45,41 @@ export const CustomToken = () => {
     symbol: string;
   }) => {
     setIsLoading(true);
-    setActiveNetwork(activeNetwork);
 
-    const contractResponse = (await contractChecker(
-      contractAddress,
-      activeNetwork.url
-    )) as ISupportsInterfaceProps;
+    try {
+      const provider = new ethers.providers.JsonRpcProvider(activeNetwork.url);
```

Comment:
@claudiovb is this correct? I don't know if we should call in the feature the ETH manager over the KeyringManager